vyxal/transpile.py

```diff
@@ -50,40 +50,41 @@ def transpile_ast(program: list[structure.Structure], indent=0) -> str:
 def transpile_single(
     token_or_struct: Union[Token, structure.Structure], indent: int
 ) -> str:
+    print("transpiling: ", token_or_struct)
     if isinstance(token_or_struct, Token):
         return transpile_token(token_or_struct, indent)
     elif isinstance(token_or_struct, structure.Structure):
         return transpile_structure(token_or_struct, indent)
     raise ValueError(
-        f"Input must be a Token or Structure,"
-        " was {type(token_or_struct).__name__}: {token_or_struct}"
+        "Input must be a Token or Structure,"
+        f" was {type(token_or_struct).__name__}: {token_or_struct}"
     )
 
 
 def transpile_token(token: Token, indent: int) -> str:
-    from helpers import indent_str, uncompress
+    from vyxal.helpers import indent_str, uncompress
 
     if token.name == TokenType.STRING:
         # Make sure we avoid any ACE exploits
-        string = uncompress(token)  # TODO: Account for -D flag
+        string = str(uncompress(token))  # TODO: Account for -D flag
         value = string.replace("\\", "\\\\").replace('"', '\\"')
         return indent_str(f'stack.append("{value}")', indent)
     elif token.name == TokenType.NUMBER:
         return indent_str(f"stack.append({token.value})", indent)
     elif token.name == TokenType.GENERAL:
         return indent_str(
-            elements.elements.get(token.value, ["pass\n", -1])[0], indent
+            elements.elements.get(token.value, ("pass\n", -1))[0], indent
         )
     elif token.name == TokenType.COMPRESSED_NUMBER:
-        return indent_str(f"stack.append({uncompress(token)})")
+        return indent_str(f"stack.append({uncompress(token)})", indent)
     elif token.name == TokenType.COMPRESSED_STRING:
-        return indent_str(f"stack.append('{uncompress(token)}')")
+        return indent_str(f"stack.append('{uncompress(token)}')", indent)
         # No need to check for ACE exploits here because this string
         # type will only ever contain lower alpha + space.
```

Comment:
famous last words...

I'd recommend also using `repr` as above.

Reply:
Except this type of string can only contain lowercase letters and spaces once uncompressed.

Reply:
I still think it's important to fix this. What if we add decompression of words with apostrophes, and then they break out of the single-quoted string? Just use repr. https://en.wikipedia.org/wiki/Defense_in_depth_(computing)

Reply:
No no I mean quite literally it's physically impossible to ACE exploit this string type.

It's `«...«`. You're confusing this type with dictionary strings.

Your point is valid for the other string type, but not this one.

Reply:
I don't care; it is good practice to apply such considerations even in situations where you think it's physically impossible